### PR TITLE
Define "Token" when necessary, or reword

### LINKED ISSUE #439: Remove references to "MISC." in conjunction with "Smart Highlighting"
Remove references to "MISC." in conjunction with "Smart Highlighting", in [THIS](https://npp-user-manual.org/docs/searching/#:~:text=sensitive%20to%20case.-,You%20activate%20smart%20highlighting,-through%20Settings%20%3E%20Preferences) area, and possibly others.

The _MISC._ subpage in the _Preferences_ is where the _Smart Highlighting_ settings used to be, but recently and currently (current = N++ v8.4.7), they are now on the _Highlighting_ subpage.

### LINKED ISSUE #440: Define the term "token"
As discussed [HERE](https://community.notepad-plus-plus.org/topic/23784/manual-chapter-search#:~:text=I%20was%20trying%20to%20find%20in%20the%20manual%20the%20definition%20of%20%E2%80%9CToken%E2%80%9D), if the word "token" is going to be used in the Notepad++ UI, as in:

image

then it should probably be defined in the UM.

### PR DESCRIPTION
**Pull request by pryrt**
- command prompt: rephrase to not use "token"
- user defined languages: rephrase to not use "token"
- preferences: in the highlighting section, define "Token" since it's part of the names of the preferences
- searching:
   - use the new **Style** menu names instead of **Mark**;
   - fix links to settings and style configurator descriptions
   - fix link name from "MISC" to "Highlighting" [closes #439]

@alankilborn, please confirm if this closes #440 and #439

### CODE REVIEW

File: content/docs/searching.md [hunk 1]
```diff
@@ -312,17 +312,17 @@ All dialog-free search actions do preserve the current search options set in the
...
 
-Note: Unfortunately, the Mark All submenu name can cause some confusion between an identically-named action button in the Mark tab of the Find family dialog.  The two types of "marking" are different but do share some features.  For example, the Copy Styled Text submenu commands will allow you to copy text that has been styled with number 1 through 5 styling OR text that has been marked using the Mark tab of Find.
+Note: In older versions of Notepad++, the **Mark All** submenu name can cause some confusion between an identically-named action button in the Mark tab of the Find family dialog.  The two types of "marking" are different but do share some features.  For example, the Copy Styled Text submenu commands will allow you to copy text that has been styled with number 1 through 5 styling OR text that has been marked using the Mark tab of Find.  This has been improved by renamnig the menu to **Style All Ocurrences of Token**.
```
Comment (alankilborn) [inline content/docs/searching.md:321]: spelling: `renamnig`

File: content/docs/searching.md [hunk 2]
```diff
@@ -312,17 +312,17 @@ All dialog-free search actions do preserve the current search options set in the
...
+You can also cause all occurrences of the word at the caret to get dynamically highlighted if you activate Smart Highlighting; the mark style then is [Settings &gt; Style Configurator &gt; Global Styles &gt; Smart Highlighting](../preferences/#global-styles). You may choose there whether the matching should be sensitive to case.
 
-You activate smart highlighting through **Settings > Preferences > MISC > Smart highlighting, Enable smart highlighting**. By default, the highlighting is case insensitive, which may be a problem sometimes. Then just toggle **Settings &gt; Preferences &gt; MISC &gt; Highlighting is case sensitive** on.  (See also [Style Configurator](../preferences/#style-configurator) and [Preferences](../preferences/#preferences).)
+You activate smart highlighting through [Settings > Preferences > Highlighting > Smart highlighting > Enable](../preferences/#highlighting). By default, the highlighting is case insensitive; if this is a problem, just toggle [Settings &gt; Preferences &gt; Highlighting &gt; Smart Highlighting > Matching: Case Sensitive](../preferences/#highlighting) on.
```
Comment (alankilborn) [inline content/docs/searching.md:325]: If you mention "case" here, should also mention "whole word"?  Fine with me if this is deemed "too heavy".

File: content/docs/searching.md (hunk 1 above)
Comment (alankilborn) [reply content/docs/searching.md]: spelling: `Ocurrences` -> 2 `c` needed

Comment (pryrt) [reply content/docs/searching.md]: fixed

File: content/docs/searching.md (hunk 2 above)
Comment (pryrt) [reply content/docs/searching.md]: reworded

File: content/docs/searching.md (hunk 1 above)
Comment (alankilborn) [reply content/docs/searching.md]: image

**Review by alankilborn — APPROVED** [review]

**Comment by alankilborn** [comment]
>please confirm if this closes https://github.com/notepad-plus-plus/npp-usermanual/issues/440 and https://github.com/notepad-plus-plus/npp-usermanual/issues/439

Looks good, for both.

**Pull request merged**